resources/lang/pt-br/beatmaps.php

```diff
@@ -21,108 +21,133 @@
-            'filtered' => 'Nenhuma discussão encontrada com este filtro.',
+            'empty' => 'Nenhuma discussão ainda!',
+            'hidden' => 'Nenhuma discussão corresponde ao filtro selecionado.',
         ],
 
         'message_hint' => [
-            'in_general' => 'Este post vai para a discussão de geral de beatmaps. Para moddar este beatmap, comece sua mensagem com um timestamp (ex. 00:12:345).',
-            'in_timeline' => 'Para moddar vários timestamps, poste várias vezes (um post por timestamp).',
+            'in_general' => 'Esta publicação irá para discussão geral de beatmaps. Para modificar este beatmap, inicie a mensagem com o timestamp (ex: 00:12:345).',
+            'in_timeline' => 'Para modificar vários timestamps, publique várias vezes (uma publicação por timestamp).',
         ],
 
-        'message_placeholder' => 'Digite aqui para postar',
-
         'message_type' => [
-            'praise' => 'Aplaudir',
+            'praise' => 'Elogio',
             'problem' => 'Problema',
             'suggestion' => 'Sugestão',
         ],
 
-        'message_type_select' => 'Escolha Tipo do Comentário',
-
         'mode' => [
             'general' => 'Geral',
-            'timeline' => 'Linha do Tempo',
+            'timeline' => 'Linha do tempo',
         ],
 
-        'require-login' => 'Faça login para postar ou responder',
-        'resolved' => 'Resolvido',
+        'new' => [
+            'timestamp' => 'Timestamp',
+            'title' => 'Nova discussão',
+        ],
 
         'show' => [
-            'title' => 'Discussão de Beatmaps',
+            'title' => ':title mapeado por :mapper',
         ],
 
         'stats' => [
             'mine' => 'Meu',
-            'pending' => 'Pendentes',
-            'praises' => 'Aplausos',
-            'resolved' => 'Resolvidos',
+            'pending' => 'Pendente',
+            'praises' => 'Elogios',
+            'resolved' => 'Resolvido',
             'total' => 'Total',
         ],
     ],
 
+    'nominations' => [
+        'disqualifed-at' => 'desqualificado :time_ago (:reason).',
+        'disqualifed_no_reason' => 'nenhuma razão especificada',
+        'disqualification-prompt' => 'Motivo da desqualificação?',
+        'disqualify' => 'Desqualificar',
+        'incorrect-state' => 'Erro ao executar essa ação, tente atualizar a página.',
+        'nominate' => 'Nomear',
+        'nominate-confirm' => 'Nomear este beatmap?',
+        'qualified' => 'Estimado para ser rankeado em :date, caso nenhum problema for encontrado.',
+        'qualified-soon' => 'Estimado para ser rankeado em breve, caso nenhum problema for encontrado.',
+        'required-text' => 'Nomeações: :current/:required',
+        'title' => 'Estado da nomeação',
+    ],
+
     'listing' => [
         'search' => [
             'prompt' => 'digite palavras-chave...',
-            'options' => 'Mais Opções de Busca',
+            'options' => 'Mais opções de buscas',
```

Comment:
should be "busca" since almost every website has it not on plural. Plus, would be more correct in portuguese as well.